lib/BattleHex.h

```diff
@@ -31,17 +31,13 @@ struct DLL_LINKAGE BattleHex //TODO: decide if this should be changed to class f
 	si16 getX() const;
 	si16 getY() const;
 	std::pair<si16, si16> getXY() const;
-	//moving to direction
-	BattleHex& moveInDir(EDir dir, bool hasToBeValid = true);
-	BattleHex& operator+=(EDir dir); //sugar for above
-	//generates new BattleHex moved by given dir
-	BattleHex movedInDir(EDir dir, bool hasToBeValid = true) const;
+	BattleHex& moveInDirection(EDir dir, bool hasToBeValid = true);
+	BattleHex& operator+=(EDir dir);
+	BattleHex generateHexMovedInDirection(EDir dir, bool hasToBeValid = true) const;
 	BattleHex operator+(EDir dir) const;
 	std::vector<BattleHex> neighbouringTiles() const;
-	//returns info about mutual position of given hexes (-1 - they're distant, 0 - left top, 1 - right top, 2 - right, 3 - right bottom, 4 - left bottom, 5 - left)
 	static signed char mutualPosition(BattleHex hex1, BattleHex hex2);
-	//returns distance between given hexes
-	static char getDistance(BattleHex hex1, BattleHex hex2);
+	static char getDistanceBetweenHexes(BattleHex hex1, BattleHex hex2);

```

Comment:
Name should explain distance calculation formula but not the fact that parameters are hexes